### PR TITLE
add magic_wormhole

### PR DESCRIPTION
**Pull request by CJ-Wright**
Here I go again

### CODE REVIEW

**Comment by CJ-Wright** [comment]
Note that this is missing the tor support, which I think may be ok for now.

**Comment by CJ-Wright** [comment]
cycle for ci

**Comment by CJ-Wright** [comment]
@conda-forge/staged-recipes I think this is ready for review (noarch package)!

File: recipes/magic_wormhole/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{% set name = "magic-wormhole" %}
+{% set version = "0.10.5" %}
+{% set sha256 = "" %}
```
Comment (ocefpaf) [inline recipes/magic_wormhole/meta.yaml]: :scream:

Comment (CJ-Wright) [reply recipes/magic_wormhole/meta.yaml]: I forgot about that, I'm glad that I'm being a good human fuzzer. Something tells me that this isn't supported behavior?

Comment (ocefpaf) [reply recipes/magic_wormhole/meta.yaml]: We should lint for that. I guess we even have an open issue for it :smile:

**Comment by CJ-Wright** [comment]
How did this pass with a typo?

**Comment by ocefpaf** [comment]
> How did this pass with a typo?

It did not. I fixed it after the failure in https://circleci.com/gh/conda-forge/staged-recipes/27038

**Comment by CJ-Wright** [comment]
Hmm, it passed without the sha and a typo at one point: https://circleci.com/gh/conda-forge/staged-recipes/27036?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Thank you!

**Pull request merged**